admin/homebrew.py

```diff
@@ -58,30 +62,18 @@ def get_class_name(version):
 
     :param str version: The version of Flocker this recipe is for.
 
-    :return str: The name of the ruby class needed if the file being created
-        is called "flocker-$VERSION.rb".
-    """
-    class_name = 'Flocker' + version
-    for disallowed_character in ['-', '.']:
-        class_name = class_name.replace(disallowed_character, '')
-    return class_name
-
-
-def get_formatted_dependency_list(dependency_graph):
-    """
-    :param tl.eggdeps.graph.Graph dependency_graph: Graph of Python
-        dependencies.
-
-    :return unicode: Space separated list of dependency names.
+    :return unicode: The name of the ruby class needed if the file being
+        created is called "flocker-$VERSION.rb".
     """
-    dependencies = []
-    for name, node in sorted(dependency_graph.iteritems()):
-        requirement = node.dist.as_requirement()
-        dependencies.append(requirement.project_name)
-    return u' '.join(dependencies)
+    class_name = list('Flocker' + version)
+    disallowed_characters = ['-', '.']
+    return u''.join([
+        character.upper() if class_name[index - 1] in disallowed_characters
+        else character for index, character in enumerate(class_name) if
+        character not in disallowed_characters])
```

Comment:
This looks correct, but is incredibly difficult to read. It should be rewritten, or at least reformated to make it clear what is going on.